blocks/footer-block/features/footer/footer.scss

```diff
@@ -139,17 +153,10 @@ footer {
 
     .footer-header {
       font-weight: bold;
-      font-size: calculateRem(14px);
-      line-height: calculateRem(21px);
-    }
-
-    .footer-item {
-      margin: map-get($spacers, 'xs') 0;
-
-      > a {
-        color: $ui-medium-gray;
-        text-decoration: none;
-      }
+      font-size: calculateRem(16px);
```

Comment:
We are setting explicit font-sizes and line-height on multiple times in this file to all be the same values - why don't we move up the declarations to the top level `footer {` and let CSS do it's thing, and only change as needed?

For the `.footer-header` you'll likely need to do the follow as they are heading elements
```
      font-size: inherit;
      line-height: inherit;
```

Reply:
cool updated ✅

Reply:
updated